src/patterns/OcAppSideBar.vue

```diff
@@ -41,6 +42,10 @@ export default {
       default: false,
       type: Boolean,
     },
+    closeButtonLabel: {
```

Comment:
Add pls docs what does this do and why is it needed. e. g. like in https://github.com/owncloud/owncloud-design-system/blob/0db4c32de7fa12a03de7b6aa0d20f09f11aa9ebd/src/elements/OCAvatar.vue#L27